micrometer/tasks/task-demo-metrics-prometheus/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.application.name=${vcap.application.name:timestamp-task}
+info.app.name=@project.artifactId@
+info.app.description=@project.description@
+info.app.version=@project.version@
+management.endpoints.web.exposure.include=health,info,bindings
```

Comment:
Don't we need some special Prometheus properties also?

Reply:
I'm assuming that those sample Tasks are meant to be used within SCDF. Later is expected to  configure the prometheus connections: 
* for local: https://dataflow.spring.io/docs/installation/local/docker-customize/#monitoring-with-prometheus-and-grafana
* for k8s: https://dataflow.spring.io/docs/installation/kubernetes/kubectl/#deploy-prometheus-and-grafana
* for CF : to be done